src/proto/h2/server.rs

```diff
@@ -364,6 +366,7 @@ impl<F, B, E> H2Stream<F, B>
 where
     F: Future<Output = Result<Response<B>, E>>,
     B: HttpBody,
+    B::Data: Send + 'static,
```

Comment:
This bound is on an impl for a `pub` item. That item is `H2Stream<F, B>`, which is used in `ConnStreamExec<F, B>`, which is also `pub` but not reachable, and used to bind `E` in the various `Connection<I, S, E>` impl blocks.